config.go

```diff
@@ -268,6 +270,9 @@ func (c *Settings) ValidateAndDefault() error {
 	if c.LogPath == "" {
 		c.LogPath = LocalLoggingOverride
 	}
+	if c.ShutdownWaitSeconds <= 0 {
```

Comment:
Should this be `< 0`? Unlike other settings, this is one we may actually want to set to 0, and the current behavior of Evergreen is as if this were set to 0.

Reply:
OK